eclass/nvidia-driver.eclass

```diff
@@ -128,50 +79,83 @@ nvidia-driver_get_mask() {
 	local nvidia_gpu drv
 
 	for nvidia_gpu in ${nvidia_gpus}; do
-		for drv in ${drv_71xx}; do
-			if [ "x${nvidia_gpu}" = "x${drv}" ]; then
-				echo "${mask_71xx}"
+		# the data below is derived from
+		# https://us.download.nvidia.com/XFree86/Linux-x86_64/396.18/README/supportedchips.html
+
+		if has ${nvidia_gpu} \
+			0020 0028 0029 002c 002d 00a0 0100 0101 0103 0150 0151 0152 0153; then
```

Comment:
I wonder if there's a way to avoid these long-ish lists of consecutive numbers. Here a simple:

```suggestion
			0020 0028 0029 002c 002d 00a0 0100 0101 0103 {0150..0153}; then
```

would work, wouldn't it?